### PR TITLE
added/adjusted below pt1

### PR DESCRIPTION
**Pull request by Ninjistix**
item.cpp - adjusted flag ref item.h
item.h - adjusted flags to match polutils
packet_system.cpp - adjusted flag ref to item.h
treasure_pool.cpp - adjusted flag ref to item.h
-- and some tab2space

### CODE REVIEW

File: src/map/treasure_pool.cpp [hunk 1]
```diff
@@ -98,55 +98,55 @@ void CTreasurePool::AddMember(CCharEntity* PChar)
...
-			break;
-		}
-	}
+    DSP_DEBUG_BREAK_IF(PChar == nullptr);
+    DSP_DEBUG_BREAK_IF(PChar->PTreasurePool != this);
+
+    //if(m_TreasurePoolType != TREASUREPOOL_ZONE){
+        //Zone drops e.g. Dynamis DO NOT remove previous lot info. Everything else does.
+        // ^ TODO: verify what happens when a winner leaves zone
+        for(int i=0; i<10; i++){
```
Comment (xdemolish) [inline src/map/treasure_pool.cpp]: space before and after operators
e.g.
```
i = 0;
i < 10;
i += 1;
```

**Comment by Ninjistix** [comment]
will check it over in a bit and adjust

**Comment by Ninjistix** [comment]
@xdemolish can you check and see if anything else needs to be adjusted

**Comment by Ninjistix** [comment]
closed by mistake -.-.. forgot to push commits to diff patch...

**Pull request closed without merging**